apps/prairielearn/src/lib/db-types.ts

```diff
@@ -657,3 +657,23 @@ export const JobSchema = z.object({
   working_directory: z.string().nullable(),
 });
 export type Job = z.infer<typeof JobSchema>;
+
+export const assessementInstanceStatsSchema = z.object({
```

Comment:
This doesn't belong in this file; `db-types` should only contain types that each correspond to a complete, single table. You should move this to the file where it's used.

Reply:
That makes sense. I have now moved this to `instructorAssessmentInstance.js` in commit [fce8031](https://github.com/PrairieLearn/PrairieLearn/pull/9072/commits/fce803136e48aa16f1aa7fbcfb1d30ca04569fe6).